### ISSUE TITLE
[ BUG ] `New-FalconIoaRule` generates `400` error when following wiki example

### ISSUE THREAD

**Issue by bk-cs** [CrowdStrike/psfalcon#399]
**Describe the bug**
Unable to create a custom IOA rule inside of of an IOA group following the [New-FalconIoaRule](https://github.com/CrowdStrike/psfalcon/wiki/New-FalconIoaRule) wiki example.

`New-FalconIoaRule` is failing to pass `field_values` as an array when submitting the rule creation request, causing a `400` response.

**Environment (please complete the following information):**
 - OS: Windows 11
 - PowerShell: 5.1
 - PSFalcon: 2.2.6

**Comment by bk-cs** [comment]
`New-FalconIoaRule` is failing to pass `field_values` as an array when submitting the rule creation request, causing a `400` response.

**Comment by bk-cs** [comment]
This bug has been fixed for the next release. If you'd like to resolve it for your local module before release, you can replace `public\ioarules.ps1` using the steps outlined below.

```powershell
Import-Module -Name PSFalcon
$ModulePath = (Show-FalconModule).ModulePath
(Invoke-WebRequest -Uri https://raw.githubusercontent.com/CrowdStrike/psfalcon/9f90b0a63a2d6196a8a380a713ca92ac26f0562d/public/ioarules.ps1 -UseBasicParsing).Content > (Join-Path (Join-Path $ModulePath public) ioarules.ps1)
```

**Comment by bk-cs** [comment]
This fix is included in the [2.2.7 release](https://github.com/CrowdStrike/psfalcon/releases/tag/2.2.7).
